helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -822,6 +828,19 @@ public void setGlobalRebalancePreference(Map<GlobalRebalancePreferenceKey, Integ
     return DEFAULT_GLOBAL_REBALANCE_PREFERENCE;
   }
 
+  /**
+   * Set the asynchronous global rebalance option.
+   * @param isAsync true if the global rebalance should be calculated asynchronously
+   */
+  public void setAsyncGlobalRebalance(boolean isAsync) {
```

Comment:
Rename to `enableAsyncGlobalRebalance`?

Reply:
I had a discussion with Junkai before, "enableXXXXXX" in our code is a bad naming. It is very confusing if you are disabling the option but you are still calling enableXXXX() method.
I would change it to be setAsyncGlobalRebalanceMode.